yarn-project/end-to-end/src/e2e_deploy_contract.test.ts

```diff
@@ -111,4 +111,25 @@ describe('e2e_deploy_contract', () => {
       );
     }
   }, 30_000);
+
+  it('it should not deploy a bad contract', async () => {
+    const goodDeploy = new ContractDeployer(TestAssertContractAbi, wallet).deploy(0);
```

Comment:
This test needs a good transaction to go through in the same block as the bad one. I deployed the same contract again but it could really be any valid transaction here.